.github/workflows/build-pages.yml

```diff
@@ -4,12 +4,9 @@
 name: Deploy Pages site
 
 on:
-  # Automatically run after the build workflow completes, since that workflow
-  # must *always* run to get PRs approved.
-  workflow_run:
-    workflows: [Build]
-    types:
-      - completed
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["community"]
```

Comment:
This should have a `paths:` filter so the workflow only runs on commits that touch the `pages/` directory I think

Reply:
Fixed :) Thanks!